src/components/PrecinctTallyReport.tsx

```diff
@@ -54,14 +60,14 @@ const PrecinctTallyReport = ({
   return (
     <Report>
       <h1>
-        <NoWrap>{precinct.name}</NoWrap> <NoWrap>{election.title}</NoWrap>{' '}
+        <NoWrap>{precinct.name}</NoWrap> <NoWrap>{election.title}</NoWrap>
```

Comment:
Please confirm that there is still a space between the election title and "Tally Report" in the rendered content.